wurst/_handles/Camera.wurst

```diff
@@ -0,0 +1,166 @@
+    SetCameraRotateMode(pos.x, pos.y, toSweep.radians(), duration)
+
+public function camerafield.set(real value, real duration)
+    SetCameraField(this, value, duration)
+
+public function camerafield.adjust(real offset, real duration)
+    AdjustCameraField(this, offset, duration)
+
+public function camerafield.get() returns real
+    return GetCameraField(this)
+
+public function camerafield.getFieldSwap(camerasetup whichSetup) returns real
+    return CameraSetupGetFieldSwap(this, whichSetup)
+
+public function camerafield.setCameraField(player whichPlayer, real value, real duration)
+    SetCameraFieldForPlayer(whichPlayer, this, value, duration)
+
+public function camerasetup.setField(camerafield whichField, real value, real duration)
+    CameraSetupSetField(this, whichField, value, duration)
+
+public function camerasetup.getField(camerafield whichField) returns real
+    return CameraSetupGetField(this, whichField)
+
+public function camerasetup.setDestPosition(vec2 pos, real duration)
+    CameraSetupSetDestPosition(this, pos.x, pos.y, duration)
+
+public function camerasetup.getDestPositionVec2() returns vec2
+    return vec2(CameraSetupGetDestPositionX(this),CameraSetupGetDestPositionY(this))
+
+public function camerasetup.apply(boolean doPan, boolean panTimed)
+    CameraSetupApply(this, doPan, panTimed)
+
+public function camerasetup.applyWithZ(real zDestOffset)
+    CameraSetupApplyWithZ(this, zDestOffset)
+
+public function camerasetup.applyForceDuration(boolean doPan, real forceDuration)
+    CameraSetupApplyForceDuration(this, doPan, forceDuration)
+
+public function camerasetup.applyForceDurationWithZ(real zDestOffset, real forceDuration)
+    CameraSetupApplyForceDurationWithZ(this, zDestOffset, forceDuration)
+
+public function camerasetup.apply(boolean doPan, player whichPlayer, real duration)
+    CameraSetupApplyForPlayer(doPan, this, whichPlayer, duration)
+
+public function camerasetup.getFieldSwap(camerafield whichField) returns real
+    return CameraSetupGetFieldSwap(whichField, this)
+
+public function player.cameraSetupApply(boolean doPan, camerasetup whichSetup, real duration)
+    CameraSetupApplyForPlayer(doPan, whichSetup, this, duration)
+
+public function player.setCameraField(camerafield whichField, real value, real duration)
+    SetCameraFieldForPlayer(this, whichField, value, duration)
+
+public function player.setCameraTargetControllerNoZ(unit whichUnit, vec2 offset, boolean inheritOrientation)
+    SetCameraTargetControllerNoZForPlayer(this, whichUnit, offset.x, offset.y, inheritOrientation)
+
+public function player.setCameraPosition(vec2 pos)
+    SetCameraPositionForPlayer(this, pos.x, pos.y)
+
+public function player.panCameraTo(vec2 pos)
+    PanCameraToForPlayer(this, pos.x, pos.y)
+
+public function player.panCameraToTimed(vec2 pos, real duration)
+    PanCameraToTimedForPlayer(this, pos.x, pos.y, duration)
+
+public function player.smartCameraPan(vec2 pos, real duration)
```

Comment:
Move single line comments into hotdoc and explain what the func does

Reply:
Done